src/visualizations/config/generators/dhis/singleValue.js

```diff
@@ -350,6 +359,7 @@ const generateDVItem = (
             formattedValue: config.formattedValue,
             subText: config.subText,
             valueColor,
+            textColor: titleColor,
```

Comment:
I'm curious if this will set the subtext color to e.g. green if the _title_ is set to green?

Reply:
`titleColor` at this point can only be the contrast color or white.
The custom title color is applied directly on the title element and not passed via `titleColor`.